content/en/docs/contribute/_index.md

```diff
@@ -0,0 +1,57 @@
+---
+content_template: templates/concept
```

Comment:
As an aside, maybe we should have a generic template because we seem to be using the concept template one for anything that is not a Task or Tutorial.